frontend/app/framework/services/analytics.service.ts

```diff
@@ -53,4 +45,31 @@ export class AnalyticsService {
             value: value
         });
     }
+
+    private configureGtag() {
+        if (this.analyticsId && this.router && this.resourceLoader && window.location.hostname !== 'localhost' ) {
+
```

Comment:
Can you just remove this line?

Reply:
which line? I don't see any highlighted lines. But if you mean "`if (analyticsId && router && resourceLoader && window.location.hostname !== 'localhost') {`" and beneath. I didn't remove it, I just cut it up in separate functions.

Reply:
which line? I don't see any highlighted lines. But if you mean "`if (analyticsId && router && resourceLoader && window.location.hostname !== 'localhost') {`" and beneath. I didn't remove it, I just cut it up in separate functions.

Reply:
The empty line after the if